services/web/client/source/class/osparc/Error.js

```diff
@@ -51,6 +51,52 @@ qx.Class.define("osparc.Error", {
     }
   },
 
+  statics: {
+    FRIENDLY_HTTP_STATUS: {
+      "200": "OK",
+      "201": "Created",
+      "202": "Accepted",
+      "203": "Non-Authoritative Information",
+      "204": "No Content",
+      "205": "Reset Content",
+      "206": "Partial Content",
+      "300": "Multiple Choices",
+      "301": "Moved Permanently",
+      "302": "Found",
+      "303": "See Other",
+      "304": "Not Modified",
+      "305": "Use Proxy",
+      "306": "Unused",
+      "307": "Temporary Redirect",
+      "400": "Bad Request",
+      "401": "Unauthorized",
+      "402": "Payment Required",
+      "403": "Forbidden",
+      "404": "Not Found",
+      "405": "Method Not Allowed",
+      "406": "Not Acceptable",
+      "407": "Proxy Authentication Required",
+      "408": "Request Timeout",
+      "409": "Conflict",
+      "410": "Gone",
+      "411": "Length Required",
+      "412": "Precondition Required",
+      "413": "Request Entry Too Large",
+      "414": "Request-URI Too Long",
+      "415": "Unsupported Media Type",
+      "416": "Requested Range Not Satisfiable",
+      "417": "Expectation Failed",
+      "418": "I'm a teapot",
```

Comment:
418 is my favorie

Reply:
TIP:  ``https://httpstatuses.com/{status_code}``  give a nice explanation of those. Might add a link to the message ?

Reply:
Too techie imo